src/getRandomNumber.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+const randomNumberOfFourDigits = Math.floor(Math.random() * 9000) + 1000;
```

Comment:
```suggestion
const randomNumberOfFourDigits = Math.floor(Math.random() * 9000) + 1000;
// need check for unique digits
```
This method can give you any 4-digit number, but you need to check if it has different digits. Here you can get 1111, but it will be incorrect.